src/libraries/restore/Directory.Build.props

```diff
@@ -2,10 +2,13 @@
   <Import Project="..\Directory.Build.props" />
 
   <PropertyGroup>
-      <!-- We need configuration-specific assets files. -->
+    <!-- We need configuration-specific assets files which requires the IntermeidateOutputPath to be set. -->
```

Comment:
Nit: typo 😄 IntermeidateOutputPath -> IntermediateOutputPath

Reply:
Nooooooooooooooooo 😭 I hope we can live the typo ^^

Reply:
haha, hopefully the build doesn't break 😄. Touching this file, I'll fix it in my PR.